test/test_manifest.json

```diff
@@ -331,12 +331,26 @@
        "lastPage": 4,
        "type": "text"
     },
+    {  "id": "taro-text-enhance",
```

Comment:
Let's add a testcase that uses `enhance` for the `tracemonkey` file too. Then way we should have horizontal, vertical and rotated text covered, as well as our default testing file, the Tracemonkey paper.